lib/sorbet-rails/tasks/rails_rbi.rake

```diff
@@ -133,12 +133,16 @@ namespace :rails_rbi do
   def generate_rbis_for_models(model_classes, available_classes)
     available_class_names = Set.new(available_classes.map { |c| c.name })
     formatted = model_classes.map do |model_class|
+      # NOTE: this logic is duplicated in SorbetRails::ModelUtils#model_class_name
```

Comment:
again, here we can use "model_class.to_s" to get the class name including the namespace.

Reply:
done